googletest/src/gtest.cc

```diff
@@ -3287,8 +3287,16 @@ static void ColoredPrintf(GTestColor color, const char* fmt, ...) {
     return;
   }
 
+#define GTEST_EMIT_ANSI_COLOR_OUTPUT              \
```

Comment:
Can we make this a function instead of a macro?

Reply:
Yes. I was initially unsure of whether a `va_list` object (i.e. the `args` here) can be passed into a function. Turns out it can, as long as:

 - it is no longer used in the calling function afterwards
 - the `va_end()` stays in the calling function, matching the `va_start()`